test/suite/quickstart/helpers.go

```diff
@@ -18,41 +23,38 @@ import (
 
 var _ = AllQuickstartsTest()
 
-var (
-	IncludeQuickstarts = os.Getenv("JX_BDD_QUICKSTARTS")
-)
-
-//CreateQuickstartTest creates a quickstart test for the given quickstart
-func CreateQuickstartTest(quickstartName string) bool {
-	return createQuickstartTests(quickstartName, false)
-}
-
-// AddAppTests Creates a jx add app test
+// AllQuickstartsTest is responsible for running `jx get quickstarts, and creating a test for each quickstart currnetly
+// available
+// Individual tests can be run with `go test test/quickstart -ginkgo.focus <quickstart name>`
 func AllQuickstartsTest() []bool {
-	if IncludeQuickstarts != "" {
-		includedQuickstartList := strings.Split(strings.TrimSpace(IncludeQuickstarts), ",")
-		tests := make([]bool, len(includedQuickstartList))
-		for _, testQuickstartName := range includedQuickstartList {
-			tests = append(tests, CreateBatchQuickstartsTests(testQuickstartName))
-		}
-		return tests
-	} else {
-		return make([]bool, 0)
+	cmd := exec.Command("jx", "get", "quickstarts")
```

Comment:
`RunWithOutput` does not work here?

Reply:
No, because we are before ginkgo is started properly or something, IIRC.

Reply:
ok